modules/securityhub/README.md

```diff
@@ -33,62 +33,51 @@ The example snippet below shows how to use this component:
 ```yaml
 components:
   terraform:
-    securityhub/common:
+    securityhub:
       metadata:
-        component: securityhub/common
+        component: securityhub
       vars:
         enabled: true
         account_map_tenant: core
         central_resource_collector_account: core-security
-        admin_delegated: false
         central_resource_collector_region: us-east-1
-        finding_aggregator_enabled: true
         create_sns_topic: true
         enable_default_standards: false
         enabled_standards:
           - standards/cis-aws-foundations-benchmark/v/1.4.0
+        admin_delegated: false
+        finding_aggregator_enabled: true
 ```
 
 ## Deployment
 
-1. Apply `securityhub/common` to all accounts
-2. Apply `securityhub/root` to `core-root` account
-3. Apply `securityhub/common` to `core-security` with `var.admin_delegated = true`
+In order to deploy AWS Security Hub to multiple accounts with single collection region.
+
+1. Deploy `securityhub` to every AWS account/region except organization root account.
+2. Deploy `securityhub` to organization root account (use `SuperAdmin` role).
+4. Update `securityhub` by designating one account as main collector account.
```

Comment:
Why does this need to be 3 steps? Shouldn't we designate the collector (in stacks YAML) and then deploy to root to do the delegation and then deploy everywhere else?